src/sites/image/postimg.cc.js

```diff
@@ -1,8 +1,11 @@
 _.register({
-  rule: [
-    'https://postimg.cc/*',
-    'https://pixxxels.cc/*',
-  ],
+  rule: {
+    host: [
+      /^postimg.cc$/,
```

Comment:
need to escape the .  in the domain parts  for all of these:
\\.cc$/,

Reply:
It that still necessary though? The code works and passed the checks without them.